app/models/conference.rb

```diff
@@ -595,6 +595,20 @@ def registration_limit_exceeded?
     registration_limit > 0 && registrations.count >= registration_limit
   end
 
+  # Returns an hexadecimal color given a collection. The returned color changed
+  # when the number of element in the collection changes and for consecutive
+  # number of elements it returns highly different colors.
+  def next_color_for_collection(collection)
+    # we have different start indices for every collection to generate a
+    # different color for every of them.
+    start_index = {
+      tracks: (program.tracks.count + 1),
+      levels: (program.difficulty_levels.count + 51),
+      types: (program.event_types.count +  101)
```

Comment:
It seems that if you don't add any white space after or before the + it complains. But if add more add one it doesn't complain.